.gitignore

```diff
@@ -17,4 +17,9 @@ exchanges/*/LICENSE
 tmp/
 dist/
 examples/*/public
-examples/*/yarn.lock
\ No newline at end of file
+examples/*/yarn.lock
+examples/*/ios/
```

Comment:
Since these are one offs I'd maybe just add them to the React Native example & create a new gitignore there 🤔

Reply:
I'd prefer them here in case we add more RN examples later, which could be the case for e.g. handling auth, etc.